README.md

```diff
@@ -47,6 +47,11 @@ Just include the starter in your pom.xml:
 </dependency>
 ```
 
+or in your build.gradle:
+
+    implementation 'org.faucet-pipeline:faucet-pipeline-spring-boot-starter:1.0.0-rc.4'
```

Comment:
```suggestion
    implementation 'org.faucet-pipeline:faucet-pipeline-spring-boot-starter:1.1.1'
```

Also with the Maven one, please.

If you know a way to use variables with MarkDown, that would be uber awesome.

Reply:
AFAIK markdown does not support variables but asciidoc does. Added an adoc version with the version being a variable. Is there any reason not to switch from md to adoc?